docs/readme/README.fr.md

```
@@ -0,0 +1,293 @@
+
+## Politique de confidentialité
+
+Consultez [PRIVACY.md](../../PRIVACY.md) pour des informations sur la façon dont DashBeam gère vos données et votre vie privée.
+
+[![Sponsor](https://img.shields.io/badge/sponsor-30363D?style=for-the-badge&logo=GitHub-Sponsors&logoColor=#EA4AAA)](https://github.com/sponsors/tonyantony300) [![Buy Me Coffee](https://img.shields.io/badge/Buy%20Me%20Coffee-FF5A5F?style=for-the-badge&logo=coffee&logoColor=FFFFFF)](https://buymeacoffee.com/tny_antny)
+
+
+## Contributeurs
+
+<a href="https://github.com/tonyantony300/dashbeam/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=tonyantony300/dashbeam" />
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)